src/contexts/ApiContext.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState, useEffect, createContext } from 'react';
+import axios from 'axios';
+import PropTypes from 'prop-types';
+import { backend } from '../conf';
+
+export const ApiContext = createContext(null);
+
+export const ApiProvider = (props) => {
+  const [poi, setPoi] = useState([]);
+  const [capsules, setCapsules] = useState([]);
+  const value = {
+    poi,
+    setPoi,
+    capsules,
+    setCapsules,
+  };
+
+  const getPoi = () => {
+    axios
+      .get(`${backend}/poi`)
+      .then((res) => {
+        setPoi(res.data);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
+
+  useEffect(() => {
+    getPoi();
+  }, []);
+
+  const getCapsules = () => {
+    axios
+      .get('http://localhost:4242/capsules')
+      .then((res) => {
+        setCapsules(res.data);
+      })
+      .catch((err) => {
+        console.log(err);
```

Comment:
API call for poi done with .env but not the capsules one

Reply:
Oh... right !